locales/es.json

```diff
@@ -14,42 +14,42 @@
 	"passenger-request-destination.provide_destination": "¿A donde desea ir? Por favor escriba la dirección exacta de la calle (Así el sistema GPS podrá entender) o describa el lugar.",
 	"order-submitted.order_submitted": "⏳ Localizando conductores... Si usted encontro un conductor, presione el botón de abajo 👇",
 	"order-submitted.cancel": "Terminar búsqueda",
-	"order-submitted.error_incorrect_input": "Opción desconocida, por favor intente de nuevo.",
+	"order-submitted.error_incorrect_input": "Opción desconocida, por favor intentelo de nuevo.",
 	"order-cancelled.order_cancelled": "👌 Gracias, terminamos de buscar conductores.",
-	"driver-select-vehicle-type.select": "Seleccione el tipo de vehículo que quiere utilizar para taxi.",
+	"driver-select-vehicle-type.select": "Seleccione el tipo de vehículo que desea pedir.",
 	"driver-select-vehicle-type.motorbike": "🏍  Motoneta / Motocicleta",
 	"driver-select-vehicle-type.car": "🚗 Automóvil",
 	"driver-select-vehicle-type.error_only_known_type": "Opción desconocida, por favor seleccione automóvil o motoneta / motocicleta.",
-	"driver-explain-checkins.text": "Casi terminamos, algunas cosas a tener en cuenta. Telegram no tiene soporte activo de localización GPS así que cada vez que cambie su ubicación realice un registro en la aplicación o registre cada hora si esta listo para aceptar nuevas ordenes de viaje. Esto puede ayudar a encontrar pasajeros cercanos a su ubicación. Presiona \"Siguiente\"",
+	"driver-explain-checkins.text": "Ya Casi hemos acabado, algunas cosas a tener en cuenta. Telegram no tiene soporte activo de localización GPS así que cada vez que cambie su ubicación realice un registro en la aplicación o registre cada hora si esta listo para aceptar nuevas ordenes de viaje. Esto puede ayudar a encontrar pasajeros cercanos a su ubicación. Presiona \"Siguiente\"",
 	"driver-explain-checkins.next": "Siguiente",
-	"driver-request-location.provide_location": "Por favor comparta su ubicación actual, así los pasajeros cercanos a usted pueden encontrarle.",
-	"driver-explain-whats-next.text": "¡Estamos listos! Será notificado sobre pasajeros y ordenes cercanas a usted. Mantén esta aplicación activa y realice registros cada hora para actualizar su ubicación. Presione \"Siguiente\"",
+	"driver-request-location.provide_location": "Por favor comparta su ubicación actual, de modo que los pasajeros cercanos a usted puedan encontrarle.",
+	"driver-explain-whats-next.text": "¡listo! Será notificado sobre pasajeros y ordenes en su cercanos a usted. Mantenga esta aplicación activa y realice registros cada hora para actualizar su ubicación. Presione \"Siguiente\"",
```

Comment:
I guess that the correct phrase should be:
`¡Listo! Usted será notificado sobre pasajeros y ordenes cercanas.`